.github/workflows/workflow.yml

```diff
@@ -11,6 +11,7 @@ jobs:
   build:
     runs-on: ${{ matrix.operating-system }}
     strategy:
+      fail-fast: false
```

Comment:
Curious question: What does `fail-fast` do?

Reply:
When fail-fast is true the other jobs in the matrix are immediately cancelled when one of the job fails. So when the MacOS job fails, the Windows job (which typically takes longer) is immediately cancelled and potentially hides different errors.